modules/appeals_api/app/swagger/appeals_api/v1/schemas/notice_of_disagreements.rb

```diff
@@ -42,8 +42,35 @@ class NoticeOfDisagreements
 
                 property :address do
                   key :type, :object
-                  key :description, 'Address of the Veteran'
-                  key :required, %i[zipCode5]
+                  key :description, 'Address of the Veteran - not required if Veteran is homeless. Cannot exceed 165
```

Comment:
I assume the `Cannot exceed 165` is because it's a total length over 5 separate fields?

Reply:
yes sir!